test-data/unit/fixtures/list.pyi

```diff
@@ -22,7 +22,8 @@ class list(Iterable[T], Generic[T]):
     def append(self, x: T) -> None: pass
     def extend(self, x: Iterable[T]) -> None: pass
 
-class tuple(Generic[T]): pass
+_T_co = TypeVar('_T_co', covariant=True)
+class tuple(Sequence[_T_co], Generic[_T_co]): pass
```

Comment:
Are you interested in fixing ``tuple`` in other fixtures? Maybe you can do this even in this PR and then turn on some asserts? (It is not necessary to inherit from ``Sequence`` everywhere, just make them generic in one variable)